dataland-frontend/src/components/pages/QualityAssurance.vue

```diff
@@ -230,6 +232,14 @@ export default defineComponent({
   methods: {
     convertUnixTimeInMsToDateString,
     humanizeString: humanizeStringOrNumber,
+    /**
+     * Converts from DataTypeEnum to GetInfoOnUnreviewedDatasetsDataTypesEnum
+     * @param input DataTypeEnum
+     * @returns GetInfoOnUnreviewedDatasetsDataTypesEnum
+     */
+    convertDataType(input: DataTypeEnum): GetInfoOnUnreviewedDatasetsDataTypesEnum {
```

Comment:
Ich schlage vor, die Funktion nicht `convert...` zu nennen, da du hier nicht konvertierst.
Du cheatest theoretisch nur, indem du dem compiler sagst 
"behandle `input` ab jetzt bitte so, als sei es vom Typ `GetInfoOnUnreviewedDatasetsDataTypesEnum`, ganz gleich ob es passt oder nicht".

Daher wäre es vermutlich besser, sie `manuallyChangeTypeOfDataTypeEnum` oder sowas.

Ein "echter" convert würde übrigens in etwa so aussehen:

```
function convertToGetInfoEnum(dataType: DataTypeEnum): GetInfoOnUnreviewedDatasetsDataTypesEnum {
    if (Object.values(GetInfoOnUnreviewedDatasetsDataTypesEnum).includes(dataType as GetInfoOnUnreviewedDatasetsDataTypesEnum)) {
        return dataType as GetInfoOnUnreviewedDatasetsDataTypesEnum;
    }
    throw new Error(`Invalid DataTypeEnum value: ${dataType}`);
}

```